.circleci/config.yml

```diff
@@ -40,9 +45,28 @@ jobs:
       - run: npm install
       - run: npm run build
 
+  deploy:
+    docker:
+      - image: circleci/node:9.11.2
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+      - run: npm install
+      - run: npm run deploy
+
 workflows:
   version: 2
   test-and-build:
     jobs:
       - test
       - build
```

Comment:
Should this step require 'test' as well (because of the dependency on the cache)?

Reply:
The cache only updates if package-lock.json changes .. So almost every run we already have a cached dependencies from last run.. 

Basically I'm trying to get as much parallelism as i can. So we don't need to make `build` wait for test. It'll use a cache if there is one, and continue a little slower if there isn't .